tests/suites/test_suite_ecp.function

```diff
@@ -1057,9 +1057,6 @@ void mbedtls_ecp_gen_key( int id )
     TEST_ASSERT_RET( mbedtls_ecp_gen_key( id, &key, &rnd_pseudo_rand,
                                           &rnd_info ), 0 );
 
-    TEST_ASSERT_RET( mbedtls_ecp_check_pubkey( &key.grp, &key.Q ), 0 );
```

Comment:
Why have these two lines been removed? It doesn't seem to have anything to do with "Adjust pk tests to TEST_ASSERT_RET".